ProcessLib/ThermoHydroMechanics/LocalAssemblerInterface.h

```diff
@@ -0,0 +1,102 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include "NumLib/Extrapolation/ExtrapolatableElement.h"
+#include "ProcessLib/LocalAssemblerInterface.h"
+
+namespace ProcessLib
+{
+namespace ThermoHydroMechanics
+{
+struct LocalAssemblerInterface : public ProcessLib::LocalAssemblerInterface,
+                                 public NumLib::ExtrapolatableElement
+{
+    virtual std::vector<double> const& getIntPtSigmaXX(
```

Comment:
I did this stress output part same with the hydromechanics process. Maybe this can be modified together with the HM process after it is merged to save some time.

Reply:
Understand. I have modified the output of the sigma and epsilon now